src/clients/archwayd/index.js

```diff
@@ -84,65 +122,28 @@ class DockerArchwayClient extends DefaultArchwayClient {
   static #getDockerArgs(archwaydHome, archwaydVersion) {
     return [
       'run',
+      '--pull',
+      'always',
```

Comment:
nice one 👍 
can we think of any case a dapp developer wouldn't want to pull it?

Reply:
I don't think so, the devs should always run the most recent version of the image